x/ethbridge/client/cli/tx.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 func parseNetworkDescriptor(networkDescriptorStr string) (oracletypes.NetworkDescriptor, error) {
```

Comment:
Since we already have verifications here, I think we should use this func where-ever we can.

Reply:
Just forget this method, will use it.